MIP113/MIP113.md

```diff
@@ -192,106 +234,133 @@ Content: CVC Creation
-* **5.4.2:** CDs must support the CVCs as much as possible, to make CVC Member participation as easy as possible by focusing on providing access to facts and data, and advice CVC Members on useful decisions that are aligned with the spirit of the CVC.
----
-* **5.4.3:** CDs may not represent Maker externally, or engage directly with counterparties or ecosystem participants, other than the CVCs. 
----
-* **5.4.4:** CDs must respect their role as guardians of the security of the Maker Protocol and act accordingly, by not taking actions that could be perceived to put the security of Maker Governance into jeopardy through politicization of the CD role.
----
-* **5.4.5:** CDs may not stand by idly as other CDs push the limits of the slippery slope of misalignment by potentially breaking the adherence requirements.
----
-* **5.4.6:** CDs may not be found to collaborate to influence Maker Governance in groups or structures outside of the CVCs.
-
-##### 5.5: Constitutional Delegates actions
+##### 5.3: Constitutional Delegate Actions
 
-CDs have formal actions that they take through cryptographically signed messages to the Maker Governance Forum.
-* **5.5.1:** CDs can choose to follow the Governance Strategy of a different CVC by posting a message that specifies a new delegate contract and a new CVC to follow. The Arbitration Facilitators must then ensure that the new Governce Strategy is displayed in the delegation frontend.
-
-##### 5.6: Constitutional Delegate participation requirements
+* **5.3.1:** CDs have formal actions that they take through cryptographically signed messages to the Maker Governance Forum.
+    * **5.3.1.1:** CDs may switch to a new Governance Strategy by posting a message that specifies a new delegate contract and a new CVC strategy to follow.
+---
+* **5.3.2:** The Arbitration Facilitators must ensure the results of any Constitutional Delegate action are represented on the Easy Governance Frontend.
 
-To maintain CD recognition a CD must have a minimum level of activity, or lose its recognition as CD and removal from *5.3.1.1A*, without being Derecognized as a Constitutional Conserver.
-* **5.6.1:** A CD must keep a minimum voting activity of 75% participation in all votes on average
-* **5.6.2:** A CD must keep a minimum communication activity of having provided comments for 75% of all votes on average.
+##### 5.4: Constitutional Delegate Income Management
 
-##### 5.7: Constitutional Delegate income management
+The top 14 CDs with the highest combined MKR delegated to their delegate contracts, who also meet minimum requirements, are eligible to receive CD income. There are two levels of CD income: Prime Constitutional Delegates (PCDs) and Reserve Constitutional Delegates (RCDs). For the purposes of calculating Constitutional Delegtate income, only MKR delegated to the two recognized contracts that are following CVC strategies is counted, MKR delegated to permissionless delegate contracts controlled by the CD is _not_ counted.
```

Comment:
```suggestion
The top 14 CDs with the highest combined MKR delegated to their delegate contracts, who also meet minimum requirements, are eligible to receive CD income. There are two levels of CD income: Prime Constitutional Delegates (PCDs) and Reserve Constitutional Delegates (RCDs). For the purposes of calculating Constitutional Delegate income, only MKR delegated to contracts listed in 5.2.4.2A is counted.
```

Reply:
There are weird edge-cases because we don't say unambiguously anywhere: "CD's may only have two active PDS contracts following CVC strategies." (also this might not be a good rule, but that's another issue)

This solves most of them cleanly, I think, so long as 5.2.4.2A is kept up-to-date.